react-app/src/native/components/SportChooser.js

```diff
@@ -0,0 +1,137 @@
+                                    source ={ require('../../images/volleyball-small.png')}
+                                />
+                            </TouchableHighlight >
+
+                        </Card>
+
+                        <Card>
+                            <TouchableHighlight  onPress={() => {
+                                sport = 3
+
+                                if (route?.params?.registerType == 'individualRegister') {
+                                    navigation.navigate('IndividualRegister', {sport:sport})
+                                
+                                } else if (route?.params?.registerType == 'reregister') {
+                                    navigation.navigate('Previousleagues', {sport:sport})
+                                
+                                } else if (route?.params?.registerType == 'newTeam') {
+                                    navigation.navigate('RegisterNewTeam', {sport:sport})
+                                } else {
+                                    navigation.navigate('RegisterNewTeam')  //Runs when the user selects it from the registration page
+                                }
+                            }}>
+                                <Image
+                                    source ={ require('../../images/football-small.png')}
+                                />
+                            </TouchableHighlight >
+                        </Card>
+
+                        <Card>
+                            <TouchableHighlight  onPress={() => {
+                                sport = 4
+
+                                if (route?.params?.registerType == 'individualRegister') {
+                                    navigation.navigate('IndividualRegister', {sport:sport})
+                                
+                                } else if (route?.params?.registerType == 'reregister') {
+                                    navigation.navigate('Previousleagues', {sport:sport})
+                                
+                                } else if (route?.params?.registerType == 'newTeam') {
+                                    navigation.navigate('RegisterNewTeam', {sport:sport})
+                                } else {
+                                    navigation.navigate('RegisterNewTeam')  //Runs when the user selects it from the registration page
+                                }
+                            }}>
+                                <Image
+                                    source ={ require('../../images/soccer-small.png')}
+                                />
+                            </TouchableHighlight >
+                        </Card>
+                    </View>
+                </Card>
+            </Content>
+        </Container>
+    )
+}
+
+const styles = StyleSheet.create({ 
+    header: {
+    fontWeight:'bold',
+    fontSize:35
+    },
+
+    subHeading: {
+        color: '#474747',
```

Comment:
Colours and font sizes should be set at the theme level so we're guaranteeing all pages use the same colours/sizes. There's a fontSizeH1 which is 27, there's brandPrimary, brandSecondary, brandLight, brandDark, basically anything that should be needed for the small amount of styles used in the app.